app/static/js/main.js

```diff
@@ -0,0 +1,26 @@
+/*jslint browser:true*/
+/*globals $*/
+var csrftoken = $('meta[name=csrf-token]').attr('content');
+
+$.ajaxSetup({
+  beforeSend: function(xhr, settings) {
+    if (!/^(GET|HEAD|OPTIONS|TRACE)$/i.test(settings.type) &&
+        !this.crossDomain) {
+      xhr.setRequestHeader("X-CSRFToken", csrftoken);
+    }
+  }
+});
+
+$(document).ready(function () {
+  // Keep track of mailto: links as connections
+  $("a[href^=mailto]").click(function (evt) {
+    var href = $(evt.target).attr('href'),
+        emails = href.split(':')[1].split('?')[0].split(',');
```

Comment:
Ah, this kind of string logic is the kind of thing that I'd love to have a [browser-side unit test suite](https://github.com/GovLab/noi2/issues/37) for!

One minor concern I have about this is: will this ever trigger on `mailto:` links that don't actually represent real users in the system? For instance, suppose we have a FAQ on the site that says "if you need help, just email help@thegovlab.org" or something. I guess it's not a big deal if such a thing happens, but one alternative might be to instead add a `data-connect-to="foo@example.org"` attribute and trigger our POST on that instead of `mailto:` links explicitly. That'd remove the need for the complex string-parsing logic above, too, but it's probably not a big deal either way.